src/workspaces/features/RunPipelineDialog/RunPipelineDialog.tsx

```diff
@@ -53,7 +53,6 @@ const VERSION_FRAGMENT = gql`
 const RunPipelineDialog = (props: RunPipelineDialogProps) => {
   const router = useRouter();
   const { pipeline, run, children } = props;
-  const [showVersionPicker, setShowVersionPicker] = useState(false);
```

Comment:
I think it's a good opportunity to revisit this form and simplify since we are adding a new parameter.
By removing this branching we keep the form unified and transparent about the values used for the run

Reply:
Thanks

We met with Mariano  and agreed to include an expandable section in the dialog to regroup Advanced options (like the one we can see in GCP console)

<img width="553" alt="Screenshot 2024-12-06 at 15 31 02-20241206-143105" src="https://github.com/user-attachments/assets/63b54a3f-552d-4170-b84f-0ac9911e7e2d">